modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerConfigurationChangesTest.java

```diff
@@ -115,178 +98,47 @@ public void setUp() {
         DistributionZonesConfiguration zonesConfiguration = clusterCfgMgr.configurationRegistry()
                 .getConfiguration(DistributionZonesConfiguration.KEY);
 
-        MetaStorageManager metaStorageManager = mock(MetaStorageManager.class);
-
-        when(metaStorageManager.appliedRevision(any())).thenReturn(completedFuture(0L));
+        vaultMgr = new VaultManager(new InMemoryVaultService());
 
         LogicalTopologyService logicalTopologyService = mock(LogicalTopologyService.class);
+        doNothing().when(logicalTopologyService).addEventListener(any());
```

Comment:
Why do you need this stubbing? It's already a mock and will do nothing

Reply:
yes, but you don't need to explicitly mock this method, this is the default behavior anyway. Or am I missing something?